travis/update-docker-hub.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
```

Comment:
Can you add `set -eo pipefail` here to stop execution if a non-zero value is returned?